src/validate.js

```diff
@@ -21,7 +21,6 @@ async function validator(options) {
 
   assert(!options.prefix, 'The `prefix` option is no longer allowed');
   assert(!options.version, 'The `version` option is no longer allowed');
-  assert(options.rootUrl, 'A `rootUrl` must be provided to taskcluster-lib-validate!');
```

Comment:
We should probably assert this isn't passed -- just to make modifying services easier..